docs/pages/eas-update/environment-variables.md

```diff
@@ -39,7 +41,7 @@ API_URL="http://localhost:3000" expo start
 
 The command above will set the `API_URL` to `"http://localhost:3000"`. Now, it's time to access that value inside our project.
 
-To access it, we can use the `expo-constants` library. It's located under the `Constants.expoConfig.extra.API_URL` property.
+To access it, we can use the `expo-constants` library. **For SDK 46 and above**, it's located at `Constants.expoConfig.extra.API_URL` property. For SDK 45 and below, it's located at `Constants.manifest.extra.API_URL` property.
```

Comment:
That doesn't seem correct. The whole thing with SDK 45 was, that it's `Constants.manifest` after build, and `Constants.manifest2` after an update.

Reply:
I think checked the wrong diff for this. Let me update this.